spectractor/extractor/spectroscopy.py

```diff
@@ -283,7 +283,7 @@ def plot_atomic_lines(self, ax, color_atomic='g', color_atmospheric='b', fontsiz
                             xycoords='axes fraction', color=color, fontsize=fontsize)
         return ax
 
-    def plot_detected_lines(self, ax=None, print_table=False):
+    def plot_detected_lines(self,output_file_name,overwrite=False,ax=None, print_table=False):
```

Comment:
Problem with arguments for other uses of the function.
 L209 , L870, spectrum.py